packages/blip/app/components/patient-data.js

```diff
@@ -337,101 +333,109 @@ class PatientDataPage extends React.Component {
-            onSwitchToDaily={this.handleSwitchToDaily}
-            onSwitchPatient={this.handleSwitchPatient}
-            onClickNavigationBack={this.handleBackToListButton}
-            canPrint={canPrint}
-            onClickPrint={this.handleClickPrint}
-          />
-        </Route>
-        <Route path={`${prefixURL}/daily`}>
-          <Daily
-            dialogDatePicker={dialogDatePicker}
-            bgPrefs={this.state.bgPrefs}
-            chartPrefs={chartPrefs}
-            dataUtil={this.dataUtil}
-            timePrefs={this.state.timePrefs}
-            patient={patient}
-            setPatient={setPatient}
-            userIsHCP={userIsHCP}
-            tidelineData={medicalData}
-            epochLocation={epochLocation}
-            msRange={msRange}
-            loading={loadingState !== LOADING_STATE_DONE}
-            canPrint={canPrint}
-            onClickPrint={this.handleClickPrint}
-            prefixURL={prefixURL}
-            onClickRefresh={this.handleClickRefresh}
-            onCreateMessage={this.handleShowMessageCreation}
-            onShowMessageThread={this.handleShowMessageThread.bind(this)}
-            onSwitchToDashboard={this.handleSwitchToDashboard}
-            onSwitchToDaily={this.handleSwitchToDaily}
-            onSwitchToTrends={this.handleSwitchToTrends}
-            onDatetimeLocationChange={this.handleDatetimeLocationChange}
-            trackMetric={this.trackMetric}
-            updateChartPrefs={this.updateChartPrefs}
-            ref={this.chartRef}
-            onSwitchPatient={this.handleSwitchPatient}
-            onClickNavigationBack={this.handleBackToListButton}
-          />
-        </Route>
-        <Route path={`${prefixURL}/trends`}>
-          <Trends
-            dialogRangeDatePicker={dialogRangeDatePicker}
-            bgPrefs={this.state.bgPrefs}
-            chartPrefs={chartPrefs}
-            dataUtil={this.dataUtil}
-            timePrefs={this.state.timePrefs}
-            epochLocation={epochLocation}
-            msRange={msRange}
-            patient={patient}
-            setPatient={setPatient}
-            userIsHCP={userIsHCP}
-            tidelineData={medicalData}
-            loading={loadingState !== LOADING_STATE_DONE}
-            canPrint={canPrint}
-            onClickPrint={this.handleClickPrint}
-            prefixURL={prefixURL}
-            onClickRefresh={this.handleClickRefresh}
-            onSwitchToDashboard={this.handleSwitchToDashboard}
-            onSwitchToDaily={this.handleSwitchToDaily}
-            onDatetimeLocationChange={this.handleDatetimeLocationChange}
-            trackMetric={this.trackMetric}
-            updateChartPrefs={this.updateChartPrefs}
-            trendsState={chartStates.trends}
-            onSwitchPatient={this.handleSwitchPatient}
-            onClickNavigationBack={this.handleBackToListButton}
-          />
-        </Route>
-      </Switch>
+      <React.Fragment>
+        <PatientNavBarMemoized
```

Comment:
Very good! I would suggest to import it as `PatientNavBar`. Most of the components from `dumb` are memoized but exported (from dumb) without the name `Memoized` in them. That way, we know than when looking for the uses of a memoized components, you don't have to search the name with `Memoized` in it.
This is just an idea, maybe I'm wrong so feel free to do whatever you want